alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/model/Monitor.kt

```diff
@@ -81,9 +82,12 @@ data class Monitor(
         sin.readLong(), // version
         sin.readString(), // name
         sin.readBoolean(), // enabled
-        Schedule.readFrom(sin),
+        Schedule.readFrom(sin), // schedule
         sin.readInstant(), // lastUpdateTime
         sin.readOptionalInstant(), // enabledTime
+        if (sin.readBoolean()) {
+            User.readFrom(sin) // user
```

Comment:
Why `User.readFrom(sin)` and not just `User(sin)` and remove that readFrom function?

Reply:
No particular reason.  const is more clean, agree.